internal/pkg/show/config/plugin/vault.go

```diff
@@ -0,0 +1,34 @@
+package plugin
+
+var VaultDefaultConfig = `tools:
+- name: vault
+  # name of the plugin
+  plugin: vault
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
+      # name of the Helm repo
+      name: hashicorp
+      # url of the Helm repo
+      url: https://helm.releases.hashicorp.com
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: hashicorp/vault
+      # release name of the chart
+      release_name: vault
+      # k8s namespace where Vault will be installed
+      namespace: devstreamtest1
```

Comment:
> After `dtm apply`, I got the pods below:
> 
> ```shell
> $ kgpo -A
> NAMESPACE        NAME                                    READY   STATUS    RESTARTS   AGE
> devstreamtest1   vault-0                                 0/1     Running   0          54s
> devstreamtest1   vault-agent-injector-7589b865fd-lbjfb   1/1     Running   0          54s
> ```
> 
> logs with vault-0 are:
> 
> ```shell
> 2022-04-19T15:12:49.637Z [INFO]  core: seal configuration missing, not initialized
> 2022-04-19T15:12:54.639Z [INFO]  core: security barrier not initialized
> 2022-04-19T15:12:54.639Z [INFO]  core: seal configuration missing, not initialized
> 2022-04-19T15:12:59.636Z [INFO]  core: security barrier not initialized
> ```

As tiexin's demands: let's valult run on k8s. do not need vault  to store secrets. 
so, if wanna this info not appear： 
1. kubectl port forward  vault svc ,and on vault dashboard, initialized vault
or : use vault command initialized  it.

Reply:
- From an end user's perspective, it's best to have these manual "TODOs" in the documentation. At least we should put some explanation about why it's not ready. And a link to HashiCorp Vault doc would help, too. Otherwise, non-experienced users will be confused, just like Daniel was.
- Although this is only a development setup, we have only 1 pod. Please make sure: there are multiple pods (for example, 3), and, if one pod is down, secrets won't be lost. So that users can test it properly.